### ISSUE TITLE
MusicGen Stereo very slow and doesnt have option to specify the length On Linux

### ISSUE THREAD

**Issue by vuaieo** [tin2tin/Pallaidium#115]
hi i realized there is no opption for MusicGen Stereo model for making it longer ,. i only can have these options: 
![MusicGen Stereo](https://github.com/user-attachments/assets/899a9dda-7eaf-43a2-a9ee-697a3a0a97f7)

and also there is some error or messege when generating: 

Model:  facebook/musicgen-stereo-medium
2024-09-02 12:03:14.101549: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:485] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-09-02 12:03:14.156430: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:8454] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-09-02 12:03:14.166339: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1452] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-09-02 12:03:14.240973: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-09-02 12:03:15.116300: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
/home/vuaieo/Desktop/UPBGE_29_8_2024/4.3/python/lib/python3.11/site-packages/torch/nn/utils/weight_norm.py:134: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
/home/vuaieo/Desktop/UPBGE_29_8_2024/4.3/python/lib/python3.11/site-packages/transformers/models/encodec/modeling_encodec.py:120: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  self.register_buffer("padding_total", torch.tensor(kernel_size - stride, dtype=torch.int64), persistent=False)
Seed: 108819309
Generate: MusicGen Stereo
Seed: 920932
`torch.nn.functional.scaled_dot_product_attention` does not support having an empty attention mask. Falling back to the manual attention implementation. This warning can be removed using the argument `attn_implementation="eager"` when loading the model.Note that this probably happens because `guidance_scale>1` or because you used `get_unconditional_inputs`. See https://github.com/huggingface/transformers/issues/31189 for more information.
Warning: 1 × Draw window and swap: 6.7849 ms, average: 6.78491592 ms
Total time: 00:00:02:831

**Comment by tin2tin** [comment]
Length fixed in: https://github.com/tin2tin/Pallaidium/commit/0b706272b7758e3b869f2aa0bce85667b2a4b762
I can't do anything about the speed afaik.
